RoboFile.php

```diff
@@ -358,6 +364,46 @@ public function qaLint() {
     return $this->_exec('./tasks/code_sniffer/vendor/bin/parallel-lint lib/ tests/ mailpoet.php');
   }
 
+  public function qaLintBuild() {
+    $task = './tasks/code_sniffer/vendor/bin/parallel-lint';
+    $filesToCheckString = implode(' ', [
+      'lib/',
+      'lib-3rd-party/',
+      'vendor/composer',
+      'vendor/mtdowling',
+      'vendor/soundasleep',
+      'vendor-prefixed/',
+      'mailpoet.php',
+    ]);
+    $filesToExcludeString = '--exclude ' . implode(' --exclude ', [
```

Comment:
Please add a code comment here that the list is taken from `build.sh`, so there will be less confusion if it needs to be updated.

Reply:
Done!